Character/Updates/Rel21/Rel21_02_004_add_securitygroup.sql

```diff
@@ -0,0 +1,4 @@
+-- Adding a security group column to the characters table:
+-- This column is used to store the character-wide security group (default is 0 for players)
+ALTER TABLE `character2`.`characters` 

```

Comment:
Non portable. 'character2' is a database name. Remove it.

Reply:
Thanks, but how would I go about fixing this?

Reply:
change the line to:
ALTER TABLE `characters` 

the database for characters will normally be pre-selected by this point based on the fact it's in the character folder.

Reply:
Thanks, I just did.